test/y2storage/clients/inst_disk_proposal_test.rb

```diff
@@ -46,36 +46,19 @@
 
     let(:partitioning_section) { {} }
 
-    context "when the Guided Setup can be shown" do
+    context "the proposal dialog" do
```

Comment:
This usage of `context` is a little bit weird. It results in something like "InstDiskProposal#run the proposal dialog does not exclude the buttons".

I would say the whole context could be removed. Something like this would make more sense to me: "InstDiskProposal#run opens the proposal dialog with all the buttons" (or maybe "...opens the proposal dialog without excluding any button").

In any case, I don't understand the need of the context (and no, changing the subject of the sentence is not what a context should be used for).